src/app.js

```diff
@@ -7,6 +7,9 @@ const csvStringify = require('csv-stringify/lib/sync');
 const json2xls = require('json2xls');
 const sendFile = require('koa-sendfile');
 const fs = require('fs');
+const { promisify } = require('util');
+
+const writeFileAsync = promisify(fs.writeFile);
```

Comment:
Node.js 10.x and upwards supports natively promises on the FS API - https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api

You should be able to do something like 

```

const fs = require('fs').promises;

await fs.writeFile(fileName, body, 'binary');
```